999-load.py

```diff
@@ -1 +1,13 @@
 from xpdan.startup.analysis import start_analysis
+from pyFAI.geometry import Geometry
+from xpdtools.calib import _save_calib_param
+from xpdan.pipelines.pipeline_utils import _timestampstr
+import time
+
+def calib_from_fit2d(directDist, centerX, centerY, 
+                     tilt=0.0, tiltPlanRotation=0.0, 
+                     pixelX=200, pixelY=200, splineFile=None):
+    g = Geometry(directDist, centerX, centerY, tilt, tiltPlanRotation, 
+                 pixelX, pixelY, splineFile))
```

Comment:
can we explicitly use `set_fit2d` method?
 https://github.com/silx-kit/pyFAI/blob/master/pyFAI/geometry.py#L1316

From the doc, there is some conversion happened in pyFAI

Reply:
oops, I meant to do that :/ (also why can't that be a base class)

Reply:
looks good to me